metadata.json

```diff
@@ -63,7 +63,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.6.1 < 6.0.0"
+      "version_requirement": ">= 4.10.9 < 6.0.0"
```

Comment:
IMHO this should be documented in the changelog so users upgrading won't miss this.

Reply:
@ekohl @bastelfreak  according https://puppet.com/misc/puppet-enterprise-lifecycle and https://puppet.com/docs/pe/2017.2/overview_version_table.html puppet 4.10.9 is the oldest still maintained puppet version. 

but yes mentioning it in the changelog would be nice.

Reply:
updated it